spec/cc/engine/analyzers/javascript/main_spec.rb

```diff
@@ -21,7 +21,7 @@
 
       expect(json["type"]).to eq("issue")
       expect(json["check_name"]).to eq("Identical code")
-      expect(json["description"]).to eq("Identical code found in 2 other locations (mass = 11)")
+      expect(json["description"]).to eq("Identical blocks of code found in 3 locations. Consider refactoring.")
```

Comment:
Yeah, this was discussed with Bryan. Now that we show the "time to fix" underneath the issue, it's less important to show the mass (higher remediation time => bigger mass). 

The mass is also still included in the "content" field.